site/_contributors/malbernaz.txt

```diff
@@ -0,0 +1,6 @@
+---
+weight: 16
```

Comment:
Similarly here, just need to update the weight to after our latest contributor :)